python/lsst/ctrl/bps/wms/htcondor/htcondor_service.py

```diff
@@ -20,17 +20,24 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 """Interface between generic workflow to HTCondor workflow system
 """
+
+__all__ = ["HTCondorService", "HTCondorWorkflow"]
+
 import os
+import re
 import logging
 from datetime import datetime, timedelta
+from pathlib import Path
 
 from lsst.ctrl.bps.bps_utils import chdir
-from lsst.ctrl.bps.wms_service import BaseWmsWorkflow, BaseWmsService
+from lsst.ctrl.bps.wms_service import BaseWmsWorkflow, BaseWmsService, WmsRunReport, WmsJobReport, WmsStates
 from lsst.ctrl.bps.generic_workflow import GenericWorkflow
-from .lssthtc import (htc_submit_dag, read_node_status, read_dag_node_log, condor_q, condor_history,
-                      HTCDag, HTCJob, htc_jobs_to_wms_report, htc_escape)
+from .lssthtc import (htc_submit_dag, read_node_status, read_dag_log, condor_q, condor_history, HTCDag,
+                      HTCJob, JobStatus, NodeStatus, htc_escape, read_dag_status, summary_from_dag,
+                      htc_check_dagman_output, MISSING_ID, pegasus_name_to_label)
 
 _LOG = logging.getLogger()
+COUNT_ERROR = -99999
```

Comment:
Doesn't look like this "constant" is a part of module's public interface. Shouldn't it start with the leading underscore then?

Reply:
Found a use case where this caused problems, so will be removing it completely.